apps/quran/models.py

```
@@ -0,0 +1,90 @@
+
+    The primary key mirrors the canonical global word index.
+    """
+
+    id = models.PositiveIntegerField(primary_key=True, help_text="Canonical global word index")
+    sura = models.ForeignKey(Sura, on_delete=models.CASCADE, related_name="words")
+    ayah = models.ForeignKey(Ayah, on_delete=models.CASCADE, related_name="words")
+    position_in_ayah = models.PositiveSmallIntegerField(help_text="Word position within its ayah (1-based)")
+    text = models.CharField(max_length=128, help_text="The word text")
+
+    class Meta:
+        ordering = ["id"]
```

Ruff (0.15.18)

[warning] Mutable default value for class attribute

(RUF012)